pytorch_lightning/loggers/mlflow.py

```diff
@@ -93,6 +95,7 @@ def __init__(
         tags: Optional[Dict[str, Any]] = None,
         save_dir: Optional[str] = './mlruns',
         prefix: str = '',
+        artifact_location: Optional[str] = None,
```

Comment:
This is, where MLFlow will save stuff, right? This does not interfer with our checkpointing?

Reply:
@justusschock Yeah, doesn't interfere with checkpointing, just allows to re-direct artifacts logged through MLFlow somewhere else. E.g. set `artifact_location` to `~/artifacts` and calls to `logger.experiment.log_artifact` save things there instead. I couldn't actually re-produce the error described in the issue, when I change `save_dir` all logs, checkpoints and artifacts move there too as expected. But I think being able to send artifacts elsewhere is still useful :)